<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainChatWindow</class>
 <widget class="QMainWindow" name="MainChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>550</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ChatCOM</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>COM port icon.png</normaloff>COM port icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextBrowser" name="textDisplay"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="inputField"/>
        </item>
        <item>
         <widget class="QPushButton" name="sendButton">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="crashBit">
          <property name="text">
           <string>CrashBit (*-*)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="collision">
          <property name="text">
           <string>Do Collision (-_-)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="isSolvedCollision">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Solved collision</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mBaudrate">
    <property name="title">
     <string>&amp;Порт</string>
    </property>
    <addaction name="mBaudrateAction"/>
   </widget>
   <widget class="QMenu" name="mDisplay">
    <property name="title">
     <string>&amp;Экран</string>
    </property>
    <addaction name="mDisplayClear"/>
   </widget>
   <addaction name="mBaudrate"/>
   <addaction name="mDisplay"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="mBaudrateAction">
   <property name="text">
    <string>&amp;Скорость передачи</string>
   </property>
   <property name="toolTip">
    <string>Изменение скорости передачи</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="mDisplayClear">
   <property name="text">
    <string>&amp;Очистить</string>
   </property>
   <property name="toolTip">
    <string>Очистить поле ввода</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
